package.json

```diff
@@ -22,20 +24,29 @@
     "test": "jest --projects packages/*",
     "bump": "lerna publish --exact --skip-npm --since \"v$(npm info @appearhere/bloom version)\"",
     "test:travis": "yarn test --coverage --maxWorkers=4",
-    "now-build": "yarn build && yarn build-storybook"
+    "now-build": "yarn build && yarn lint && yarn build-storybook",
+    "lint": "cd packages/core && flow"
```

Comment:
Can you name this as `lint-flow` or `flow`?

Reply:
ah yes good shout @zlahham. Updated!